classes/config/Config.php

```diff
@@ -29,6 +29,39 @@
 
 class Config
 {
+    /**
+     * The sensitive data from the config files in the formate of `section` to `keys` mapping as
+     * [
+     *   'section1' => ['key1', 'key2', ...],
+     *   'section2' => ['key1', 'key2', ...],
+     * ]
+     */
+    public const SENSITIVE_DATA = [
+        'database' => [
+            'password',
+        ],
+        'email' => [
+            'smtp_password',
+        ],
+    ];
+
+    /**
+     * Check and determine if the given section key is sensitive data or not
+     *
+     * @param string $section
+     * @param string $key
+     *
+     * @return bool
+     */
+    public static function isSensitive($section, $key)
```

Comment:
For `stable-3_4_0` and `main`, use explicit typehints in the function declaration rather than PHPdoc-style comments (unless the comments add anything beyond the typehints, in which case use both). For porting back to `stable-3_3_0`, don't use typehints yet.

Reply:
Added. I know about this but intentionally did not add the type hint as none of the other method has it and seems odd to have only one method to have it . 

perhaps in one of other issue/PR we need to update the whole code base to have type hints to all places which possible using the [rector](https://github.com/rectorphp/rector) .